snappy_wrappers/wrappers/somatic_variants_checking/summarize-vcf.py

```diff
@@ -0,0 +1,264 @@
+# import packages
+import argparse
+import gzip
+import json  # for writing out the json file
+
+from cyvcf2 import VCF  # for reading vcf file
+import tabix
+
+
+def check_variant_in_bed(v_chrom, v_start, v_end, bed_intervals, padding=0):
+    try:
+        tmp = []
+        records = bed_intervals.query(v_chrom, v_start - int(padding), v_end + int(padding))
+        for record in records:
+            tmp.append(record)
+        if len(tmp) >= 1:
+            return True
+        else:
+            return False
+    except OSError:
+        print("You should index the bed file with tabix first!")
+
+
+def get_contigs_from_bed_file(bedfile):
+    contigs = []
+    try:
+        with gzip.open(bedfile, "r") as intervals:
+            for interval in intervals:
+                line = interval.decode()
+                tmp = line.split("\t")[0]
+                if tmp not in contigs:
+                    contigs.append(tmp)
+        return contigs
+    except OSError:
+        print("Error reading the GZIP file.")
+
+
+def get_variant_type(ref, alt):
+    if (len(alt) == 1) and (len(ref) == len(alt)):
+        return "snp"
+    elif len(alt) != len(ref):
+        return "indels"
+
+
+def check_sp_read(variant, minimal, limited):
+    dp = variant.format("AD")[1][1]
```

Comment:
How are you making sure that the second `FORMAT` column is actually the tutor sample?